src/interfaces/observations.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose';
+import { User } from 'src/interfaces/user';
+
+export interface Observation {
+  title: string;
+  description: string;
+  type: string;
```

Comment:
Let's create a type for the type of observation it would be (private or public)

```
  export type ObservationType: 'PUBLIC' | 'PRIVATE';
```

```suggestion
  type: ObservationType;
```